src/spatialdata/_core/spatialdata.py

```diff
@@ -931,195 +879,557 @@ def transform_to_coordinate_system(
                 elements[element_type][element_name] = transformed
         return SpatialData(**elements, tables=sdata.tables)
 
-    def write(
+    def describe_elements_are_self_contained(self) -> dict[str, bool]:
```

Comment:
Minor nitpick: I'm not a huge fan of this name. Perhaps something more like `elements_are_self_contained()`